src/app/client/client-management/client-management.component.ts

```
@@ -0,0 +1,17 @@
+import { InviteEmployeeComponent } from '../invite-employee/invite-employee.component';
+import { EmployeeListComponent } from '../employee-list/employee-list.component';
+
+@Component({
+  selector: 'app-client-management',
+  standalone: true,
+  templateUrl: './client-management.component.html',
+  styleUrls: ['./client-management.component.scss'],
+  imports: [EmployeeListComponent, InviteEmployeeComponent, ClientInfoComponent],
+})
+export class ClientManagementComponent implements OnInit {
+  constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)